platform/pulpcore/app/urls.py

```diff
@@ -63,6 +63,7 @@ def router_for_nested_viewset(viewset):
 
 
 urlpatterns = [
+    url(r'^content/', ContentView.as_view()),
```

Comment:
Is it possible to use `ContentView.BASE_URL` instead? Seems undesirable to define a constant but hardcode here.